index.html

```diff
@@ -244,6 +244,67 @@ <h4>LET'S CONNECT</h4>
           </div>
         </div>
       </section>
+
+      <!-- contact section -->
+      <section class="contact-section align-center padding">
+        <div>
+          <h1>CONTACT US</h1>
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+        <div>
+          <form action="https://formspree.io/f/xoqypwwp" method="POST">
+            <input
+              class="full-width input-padding"
+              type="email"
+              name="name"
+              placeholder="Text Input"
+              minlength="30"
+              required
+            />
+            <input
+              class="full-width input-padding"
+              type="email"
+              name="email"
+              placeholder="jeremy@gmail.com"
+              required
+            />
+            <textarea
+              class="full-width input-padding"
+              name="message"
+              placeholder="Enter text here"
+              minlength="500"
+            ></textarea>
```

Comment:
- [ ] Please add the `required` attribute so we do not send an empty message as well 🙏